<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="WKS22218" name="com.integration.testcases.IntegrationTests" tests="1" failures="0" timestamp="06 Feb 2018 17:26:53 GMT" time="167.764" errors="1">
  <testcase name="test_Connecture_M360_Integration" time="167.764" classname="com.integration.testcases.IntegrationTests">
    <error type="com.jacob.com.ComFailException" message="Can&#039;t get object clsid from progid">
      <![CDATA[com.jacob.com.ComFailException: Can't get object clsid from progidat com.jacob.com.Dispatch.createInstanceNative(Native Method)at com.jacob.com.Dispatch.<init>(Dispatch.java:99)at com.jacob.activeX.ActiveXComponent.<init>(ActiveXComponent.java:58)at autoitx4java.AutoItX.<init>(AutoItX.java:181)at com.m360.pages.IntegrationFlows.searchAndValidateDayNMinus1PendingEnrollmentsInM360(IntegrationFlows.java:67)at com.integration.testcases.IntegrationTests.validate_Enrollment_In_M360_Day_N_Minus_1(IntegrationTests.java:59)at com.integration.testcases.IntegrationTests.test_Connecture_M360_Integration(IntegrationTests.java:42)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- test_Connecture_M360_Integration -->
</testsuite> <!-- com.integration.testcases.IntegrationTests -->
